torch/csrc/jit/codegen/cuda/root_domain_map.cpp

```diff
@@ -118,7 +118,7 @@ std::unordered_map<IterDomain*, IterDomain*> PairwiseRootDomainMap::map(
     // non-broadcast domains, except with a broadcast producer and a
     // consumer trivial reduction
     if (is_exact_ && producer_id->isBroadcast() != consumer_id->isBroadcast() &&
-        !(producer_id->isBroadcast() && consumer_id->isTrivialReduction())) {
+        !(producer_id->isBroadcast() && consumer_id->isReduction())) {
```

Comment:
I think here we should just remove the condition of `!(producer-id->isBroadcast() && consumer_id->isReduction())`. This was to allow mappings of producer broadcasts and consumer trivial reductions, which shouldn't happen anymore.

Reply:
Looks like I need to convert trivial reductions to squeeze in Welford before I can remove this. Will do it.